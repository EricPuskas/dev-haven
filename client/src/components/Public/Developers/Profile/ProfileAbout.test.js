import { shallow } from "enzyme";
import React from "react";
import ProfileAbout from "./ProfileAbout";

it("expect to render Profile About component", () => {
  const mockProps = {
    profile: {
      bio: "Lorem Ipsum",
      skills: [1, 2, 3, 4, 5],
      user: { name: "John" }
    }
  };
  expect(shallow(<ProfileAbout {...mockProps} />).debug()).toMatchSnapshot();
});
