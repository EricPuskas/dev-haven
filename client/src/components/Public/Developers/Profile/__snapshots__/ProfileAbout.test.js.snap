// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Profile About component 1`] = `
"<div className=\\"profile-about bg-light p-2\\">
  <h2 className=\\"text-primary\\">
     
    John
    &#39;s Bio 
  </h2>
  <p>
    Lorem Ipsum
  </p>
  <div className=\\"line\\" />
  <h2 className=\\"text-primary\\">
    Skill Set
  </h2>
  <div className=\\"skills\\">
    <div className=\\"p-1\\">
      <i className=\\"fas fa-check\\" />
       
      1
    </div>
    <div className=\\"p-1\\">
      <i className=\\"fas fa-check\\" />
       
      2
    </div>
    <div className=\\"p-1\\">
      <i className=\\"fas fa-check\\" />
       
      3
    </div>
    <div className=\\"p-1\\">
      <i className=\\"fas fa-check\\" />
       
      4
    </div>
    <div className=\\"p-1\\">
      <i className=\\"fas fa-check\\" />
       
      5
    </div>
  </div>
</div>"
`;
